src/services/pathContents/commit/file/usePrefetchCommitFileEntry.spec.tsx

```diff
@@ -250,9 +254,9 @@ describe('usePrefetchCommitFileEntry', () => {
 
         await result.current.runPrefetch()
 
-        await waitFor(() => expect(mockVars).toHaveBeenCalled())
+        await waitFor(() => expect(mockVars).toBeCalled())
```

Comment:
Just to note that `toBeCalled` has been deprecated, and they suggest using `toHaveBeenCalled` instead, same for other `toHaveBeen*` matchers.